standalone-reactui/src/test/java/io/atlasmap/standalone/E2ETest.java

```diff
@@ -0,0 +1,143 @@
+        String cwd = System.getProperty("user.dir");
+        fileInput.sendKeys(cwd + "/src/test/resources/json-schema-source-to-xml-schema-target.adm");
+        waitForLoad.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//article[@aria-label='JSONSchemaSource']")));
+        WebElement orderBtn = driver.findElement(By.xpath("//button[@id='sources-field-atlas:json:JSONSchemaSource:source:/order-toggle']"));
+        orderBtn.click();
+        WebElement addrBtn = driver.findElement(By.xpath(
+            "//button[@id='sources-field-atlas:json:JSONSchemaSource:source:/order/address-toggle']"));
+        addrBtn.click();
+        WebElement addrDiv = driver.findElement(By.xpath(
+            "//button[@id='sources-field-atlas:json:JSONSchemaSource:source:/order/address-toggle']/../.."));
+        WebElement cityDiv = addrDiv.findElement(By.xpath(".//button[@data-testid='grip-city-button']/../../../.."));
+        Actions action = new Actions(driver);
+        action.moveToElement(cityDiv).perform();
+        WebElement showDetailsBtn = cityDiv.findElement(By.xpath(".//button[@data-testid='show-mapping-details-button']"));
+        showDetailsBtn.click();
+        WebElement detailsCity = waitForLoad.until(ExpectedConditions.visibilityOfElementLocated(By.xpath(
+            "//div[@role='dialog']//div[@aria-labelledby='mapping-field-city']")));
+        assertNotNull(detailsCity);
+
+        WebElement exportBtn = driver.findElement(By.xpath("//button[@data-testid='export-mappings-button']"));
+        exportBtn.click();
+        WebElement dialogDiv = driver.findElement(By.xpath("//div[@role='dialog' and @aria-label='Export Mappings and Documents.']"));
+        WebElement exportInput = dialogDiv.findElement(By.xpath(".//input[@aria-label='Export Mappings and Documents.']"));
+        String exportAdmFileName = UUID.randomUUID().toString() + "-exported.adm";
+        exportInput.sendKeys(exportAdmFileName);
+        WebElement confirmBtn = dialogDiv.findElement(By.xpath(".//button[@data-testid='single-input-dialog-confirm-button']"));
+        WatchService watcher = FileSystems.getDefault().newWatchService();
+        Path dirPath = Paths.get(DLDIR);
+        dirPath.register(watcher, StandardWatchEventKinds.ENTRY_CREATE);
+        confirmBtn.click();
+        WatchKey key = watcher.poll(5, TimeUnit.SECONDS);
+        for (WatchEvent<?> event : key.pollEvents()) {
+            if (!StandardWatchEventKinds.ENTRY_CREATE.name().equals(event.kind().name())) {
+                continue;
+            };
+            Path eventPath = (Path)event.context();
+            if (!exportAdmFileName.equals(eventPath.getFileName().toString())) {
+                continue;
+            }
+            ADMArchiveHandler handler = new ADMArchiveHandler(getClass().getClassLoader());
+            handler.load(Paths.get(DLDIR + File.separator + exportAdmFileName));
+            assertEquals("UI.0", handler.getMappingDefinition().getName());
```

Comment:
verify downloaded .adm file can be extracted and parsing mapping definition file succeeds